google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/data/v2/stub/EnhancedBigtableStub.java

```diff
@@ -736,12 +736,22 @@ public Map<String, String> extract(MutateRowsRequest mutateRowsRequest) {
     ServerStreamingCallable<MutateRowsRequest, MutateRowsResponse> withStatsHeaders =
         new StatsHeadersServerStreamingCallable<>(base);
 
+    ServerStreamingCallable<MutateRowsRequest, MutateRowsResponse> rateLimitingStreamingCallable =
+        null;
```

Comment:
this is a bit ugly. I think it might be easier to follow by having a local callable variable that just get reassigned in each step:

ServerStreamingCallable<MutateRowsRequest, MutateRowsResponse> callable = GrpcRawCallableFactory.createServerStreamingCallable(...);

callable = new StatsHeadersServerStreamingCallable<>(callable);

if (settings.bulkMutateRowsSettings().isServerInitiatedFlowControlEnabled()) {
 callable = new RateLimitingServerStreamingCallable(callable);
}

callable = new ConvertExceptionCallable<>(callable...); 
...